.github/pull_request_template.md

```diff
@@ -0,0 +1,21 @@
+Thanks for your contribution and we appreciate it a lot.
```

Comment:
This template is quite complex and may discourage contributions. We can just keep necessary items.

Reply:
I add optional in the title, is it OK?

Reply:
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.